src/main/kotlin/org/dropProject/controllers/ApplicationContextListener.kt

```diff
@@ -119,7 +125,8 @@ class ApplicationContextListener(val assignmentRepository: AssignmentRepository,
             val assignment = Assignment(id = "sampleJavaProject", name = "Sample Java Assignment",
                     packageName = "org.dropProject.samples.sampleJavaAssignment", ownerUserId = "teacher1",
                     submissionMethod = SubmissionMethod.UPLOAD,
-                    gitRepositoryUrl = "git@github.com:palves-ulht/sampleJavaAssignment.git",
+                    //gitRepositoryUrl = "git@github.com:palves-ulht/sampleJavaAssignment.git",
+                    gitRepositoryUrl = "git@github.com:brunompc/sampleJavaAssignment.git",
```

Comment:
I don't feel confortable with this dependency. In fact, I don't feel confortable with the dependency on palves-ulht repo either. There is an "official" repo for the sampleJavaAssignment on https://github.com/drop-project-edu/sampleJavaAssignment. Can you (1) make a PR of your changes on the sampleJavaAssignment repo and (2) change the url on this line?

Reply:
Done.